<ui version="4.0" >
 <class>OMachinesDialog</class>
 <widget class="QDialog" name="OMachinesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>217</height>
   </rect>
  </property>
  <property name="baseSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string> Ip o Nombre</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="pushButton_4" >
     <property name="text" >
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="OMachinesLineEdit" />
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="OMachineAddPushButton" >
     <property name="text" >
      <string>AÃ±adir</string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="2" column="0" >
    <widget class="QTableView" name="OMachinesTableView" />
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="OMachineRemovePushButton" >
     <property name="text" >
      <string>Borrar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="OClosePushButton" >
     <property name="text" >
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Icono Aqui</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>OMachinesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>205</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>218</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
